streaming/src/main/java/org/openmrs/analytics/FhirStreaming.java

```diff
@@ -26,14 +26,33 @@
  */
 public class FhirStreaming {
 	
-	private static final Logger log = LoggerFactory.getLogger(FhirStreaming.class);
+	private static final Logger log = LoggerFactory.getLogger(FhirEventWorker.class);
+	
+	// TODO: set as arg or env variable? using constant for simplicity
+	private static final String FEED_ENDPOINT = "/ws/atomfeed";
```

Comment:
What are the changes in this file? Are they related to BasicAuth? If not, please split them into a separate PR (and possibly create an issue for it too).

Reply:
At least some of these changes are required to be able to pass the BasicAuth username and password using command line args. I can take out the url constants, but we need to create both an atomfeed url and fhir endpoint url from the same base openmrs url.